bcda/bcdacli/cli.go

```diff
@@ -598,19 +610,35 @@ func archiveExpiring(hrThreshold int) error {
 	for _, j := range jobs {
 		t := j.UpdatedAt
 		elapsed := time.Since(t).Hours()
-		if int(elapsed) >= hrThreshold {
-
-			id := int(j.ID)
-			jobDir := fmt.Sprintf("%s/%d", os.Getenv("FHIR_PAYLOAD_DIR"), id)
-			expDir := fmt.Sprintf("%s/%d", os.Getenv("FHIR_ARCHIVE_DIR"), id)
-
-			err = os.Rename(jobDir, expDir)
+		id := int(j.ID)
```

Comment:
I don't think we need to cast the j.ID (uint)to an int. Sprintf will handle all numeric types.